src/Http/Routing/src/LinkGeneratorEndpointNameAddressExtensions.cs

```diff
@@ -66,16 +66,67 @@ public static class LinkGeneratorEndpointNameAddressExtensions
     /// Generates a URI with an absolute path based on the provided values.
     /// </summary>
     /// <param name="generator">The <see cref="LinkGenerator"/>.</param>
+    /// <param name="httpContext">The <see cref="HttpContext"/> associated with the current request.</param>
     /// <param name="endpointName">The endpoint name. Used to resolve endpoints.</param>
     /// <param name="values">The route values. Used to expand parameters in the route template. Optional.</param>
-    /// <param name="pathBase">An optional URI path base. Prepended to the path in the resulting URI.</param>
+    /// <param name="pathBase">
+    /// An optional URI path base. Prepended to the path in the resulting URI. If not provided, the value of <see cref="HttpRequest.PathBase"/> will be used.
+    /// </param>
     /// <param name="fragment">An optional URI fragment. Appended to the resulting URI.</param>
     /// <param name="options">
     /// An optional <see cref="LinkOptions"/>. Settings on provided object override the settings with matching
     /// names from <c>RouteOptions</c>.
     /// </param>
     /// <returns>A URI with an absolute path, or <c>null</c>.</returns>
     [SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters", Justification = "Required to maintain compatibility")]
```

Comment:
This is a new overload. How is it "Required to maintain compatibility"❔ Same for a few other _new_ overloads in this `class`.

Reply:
I'm not sure. That comment is used when suppressing all warnings of this type - https://github.com/dotnet/aspnetcore/search?p=1&q=RS0026